Packages/com.unity.inputsystem/Tests/TestFixture/InputTestFixture.cs

```diff
@@ -90,6 +90,12 @@ public virtual void Setup()
 
                 runtime = new InputTestRuntime();
 
+#if UNITY_INPUT_SYSTEM_ENABLE_GLOBAL_ACTIONS_API
+                // we want to run most tests without the high level API being enabled because it interferes with
```

Comment:
[CRITICAL] This 'interfering with global states' is concerning!

This seems a risk to landing it / shipping